yt/frontends/stream/data_structures.py

```diff
@@ -26,6 +26,7 @@
 
 from numbers import Number as numeric_type
 
+from cykdtree import PyKDTree
```

Comment:
You'll want to swap this import to the new vendored cykdtree:
from yt.utilities.lib.cykdtree import PyKDTree